web/src/components/ToolbarItem.tsx

```diff
@@ -23,14 +25,36 @@ type ItemProps =
 type ToolbarItemProps = {
   icon: string
   text: string
+  isDisabled?: boolean
 } & ItemProps
 
-const ToolbarItem = ({ href, text, icon, onClick }: ToolbarItemProps): ReactElement => (
-  // @ts-expect-error wrong types from polymorphic 'as', see https://github.com/styled-components/styled-components/issues/4112
-  <StyledToolbarItem as={onClick ? Button : undefined} href={href} onClick={onClick} label={text}>
-    <StyledIcon src={icon} />
-    <StyledSmallViewTip>{text}</StyledSmallViewTip>
-  </StyledToolbarItem>
-)
+const ToolbarItem = ({ href, text, icon, isDisabled = false, onClick }: ToolbarItemProps): ReactElement => {
+  const { t } = useTranslation('categories')
+  return isDisabled ? (
+    <Tooltip text={t('disabledPdf')} flow='up'>
+      <StyledToolbarItem
+        as={onClick ? Button : undefined}
+        // @ts-expect-error wrong types from polymorphic 'as', see https://github.com/styled-components/styled-components/issues/4112
+        href={null}
+        onClick={onClick}
+        label={text}
+        disabled={isDisabled}>
+        <StyledIcon src={icon} disabled={isDisabled} />
+        <StyledSmallViewTip>{text}</StyledSmallViewTip>
+      </StyledToolbarItem>
+    </Tooltip>
+  ) : (
+    <StyledToolbarItem
+      as={onClick ? Button : undefined}
+      // @ts-expect-error wrong types from polymorphic 'as', see https://github.com/styled-components/styled-components/issues/4112
+      href={href}
+      onClick={onClick}
+      label={text}
+      disabled={isDisabled}>
+      <StyledIcon src={icon} disabled={isDisabled} />
+      <StyledSmallViewTip>{text}</StyledSmallViewTip>
+    </StyledToolbarItem>
+  )
+}
```

Comment:
🔧 This can be simplified by turning the `StyledToolbarItem` part into a variable that is used twice.